plugins/ag-charts-generate-example-files/src/executors/generate/generator/utils/getDarkModeSnippet.ts

```diff
@@ -1,3 +1,4 @@
+// NOTE: This is duplicated from `external/ag-website-shared/src/utils/getDarkModeSnippet.ts`
```

Comment:
I needed to duplicate this in `ag-website-shared` because I can't load files outside of plugins here